src/App.vue

```diff
@@ -104,6 +104,31 @@ export default {
 				},
 			]),
 			link: [
+				// Fonts in format woff2nt'
+				{
+					rel: 'preload',
+					href: require('@/assets/fonts/PostGrotesk-BoldItalic.woff2'),
+					crossorigin: 'anonymous',
+					as: 'font'
+				},
+				{
+					rel: 'preload',
+					href: require('@/assets/fonts/PostGrotesk-Book.woff2'),
+					crossorigin: 'anonymous',
+					as: 'font'
+				},
+				{
+					rel: 'preload',
+					href: require('@/assets/fonts/PostGrotesk-BookItalic.woff2'),
+					crossorigin: 'anonymous',
+					as: 'font'
+				},
+				{
+					rel: 'preload',
+					href: require('@/assets/fonts/PostGrotesk-Medium.woff2'),
+					crossorigin: 'anonymous',
+					as: 'font'
+				},
```

Comment:
As I mentioned offline, @emuvente I'm a little concerned about how much extra time preloading "all" fonts will add to the first page load for our site visitors...